### ISSUE TITLE
Please make dudestar more accessible with sscreenreaders

### ISSUE THREAD

**Issue by SM6XGP** [nostar/dudestar#57]
When using a screenreader I discovered following issues:
When navigate with the keyboard around the interface the screen reader don't tell me the correct labels for the controlls. It tell me there is a edit box, checkbox or combobox but not the correct label.
Example is the field for talkgroup entry on dmr, screenreader says callsign on that field.
Hope that can be fixed in a future release.

**Comment by nostar** [comment]
Older version of dudestar had only a single pane and everything was jammed into the UI in a disorganized matter, with some edit fields not labelled.  The latest version has a tabbed UI with a more organzied layout.  Maybe that will make your screenreader work better.

**Comment by nostar** [comment]
Closing
